### PR TITLE
Add additional pt translations

### PR DESCRIPTION
**Pull request by gfpacheco**
#### Description of changes

Add additional pt translations

#### Description of how you validated changes

I didn't, just followed the example from the es translations

#### Checklist

- [x] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [x] PR description included
- [x] Relevant documentation is changed or added (and PR referenced)
- [x] `yarn test` passes and tests are updated/added
- [x] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by zchenwei — DISMISSED** [review]
LGTM

File: packages/ui/src/i18n/dictionaries/authenticator/pt.ts [hunk 1]
```diff
@@ -3,13 +3,16 @@ import { AuthenticatorDictionary } from './types';
...
+  'Add your Profile': 'Add your Profile',
+  'Add your Website': 'Add your Website',
   'Back to Sign In': 'Voltar para Entrar',
   'Change Password': 'Mudar senha',
   Changing: 'Mudando',
   Code: 'Código',
   'Confirm Password': 'Confirme a Senha',
   'Confirm Sign Up': 'Confirmar inscrição',
   'Confirm SMS Code': 'Confirme o código SMS',
+  'Confirm MFA Code': 'Confirm MFA Code',
```
Comment (esauerbo) [inline packages/ui/src/i18n/dictionaries/authenticator/pt.ts:15]: Should we look up these translations as well? seems like it was a miss

Comment (gfpacheco) [reply packages/ui/src/i18n/dictionaries/authenticator/pt.ts:15]: Oh I missed translating this one! Will fix asap!

**Review by zchenwei — DISMISSED** [review]

File: packages/ui/src/i18n/dictionaries/authenticator/pt.ts [hunk 2]
```diff
@@ -3,13 +3,16 @@ import { AuthenticatorDictionary } from './types';
 export const ptDict: AuthenticatorDictionary = {
   'Account recovery requires verified contact information':
     'A recuperação da conta requer informações de contato verificadas',
+  'Add your Profile': 'Add your Profile',
+  'Add your Website': 'Add your Website',
```
Comment (esauerbo) [inline packages/ui/src/i18n/dictionaries/authenticator/pt.ts:7]: Could you do these two as well @gfpacheco 😅

**Review by esauerbo — APPROVED** [review]

**Review by zchenwei — APPROVED** [review]

**Pull request merged**